Public/Src/FrontEnd/MsBuild/PipConstructor.cs

```diff
@@ -570,40 +600,14 @@ private bool TryAddMsBuildArguments(ProjectWithPredictions project, PipDataBuild
             }
 
             // Targets to execute.
-            // If the prediction is available, there should be at least one target (otherwise it makes no sense to schedule the project, and we should have caught this earlier)
-            // If the prediction is not available, we fallback to call default targets (which means not passing any specific /t:). A more strict policy would be to bail out
-            // here saying that the project is not complying to the target protocol specification. We leave it relaxed for now, but we log it.
-            // https://github.com/Microsoft/msbuild/blob/master/documentation/specs/static-graph.md
-            if (project.PredictedTargetsToExecute.IsPredictionAvailable)
-            {
-                var targets = project.PredictedTargetsToExecute.Targets;
-                Contract.Assert(targets.Count > 0);
-                foreach (string target in targets)
-                {
-                    pipDataBuilder.Add(PipDataAtom.FromString($"/t:{target}"));
-                }
-            }
-            else
+            var targets = project.PredictedTargetsToExecute.Targets;

```

Comment:
Where are the default targets being added if any of the referencing projects do not specify the protocol? Can't find it.

Reply:
Ah, it's happening here: https://github.com/microsoft/BuildXL/pull/293/files#diff-033a479d4db36673974a933ddc616049R453